workflow-job

```diff
@@ -179,13 +180,20 @@ def libPathJPA = '/jpa/destination/james-server-jpa-guice.lib'
 def cliJarPathJPA = '/jpa/destination/james-server-cli.jar'
 def clilibPathJPA = '/jpa/destination/james-server-cli.lib'
 
-def zipPath = '/spring/destination/james-server-app-3.2.0-SNAPSHOT-app.zip'
-
+def jamesVersion = jamesVersion()
+def zipPath = "/spring/destination/james-server-app-${jamesVersion}-app.zip"
 def keystorePath = '/keys/keystore'
 def jamesCliWithOptions = 'java -jar /root/james-cli.jar -h 127.0.0.1'
-def jamesCliWithOptionsSpring = '/root/james-server-app-3.2.0-SNAPSHOT/bin/james-cli.sh -h 127.0.0.1 -p 9999'
+def jamesCliWithOptionsSpring = "/root/james-server-app-${jamesVersion}/bin/james-cli.sh -h 127.0.0.1 -p 9999"
 def useMavenCache = '-v maven-cache-volume:/root/.m2'
 
+private String jamesVersion() {
+    if (version == null) {
+        return "3.2.0-SNAPSHOT";
```

Comment:
what about throwing instead?

Reply:
No, this will be used for non branches builds (only PR yet).

Reply:
ok, I was complaining that it makes us modify the build for each new release, but in fact we already need to do this...